src/helm/benchmark/slurm_runner.py

```diff
@@ -203,21 +186,74 @@ def cancel_all_jobs():
             failed_runs_str = ", ".join([f'"{run_name}"' for run_name in failed_run_names])
             raise RunnerError(f"Failed runs: [{failed_runs_str}]")
 
+    def _submit_slurm_job_for_run_spec(self, run_spec: RunSpec) -> int:
+        """Create a Slurm job for the RunSpec and return the Slurm job ID."""
+        # Create a worker Slurm job that reads from the SlurmRunnerSpec and RunSpec files
+        run_name = run_spec.name
+        run_spec_json = to_json(run_spec)
+        run_spec_path = os.path.join(self.run_specs_dir, f"{run_name}.json")
+        hlog(f"Writing RunSpec for run {run_name} to {run_spec_path}")
+        write(file_path=run_spec_path, content=run_spec_json)
+
+        log_path = os.path.join(self.logs_dir, f"{run_name}.log")
+        # Requires that SlurmRunnerSpec has already been written to self.slurm_runner_spec_path.
+        # It should have been written at the start of self.run_all()
+        command = (
+            f"{sys.executable}"
+            f" -m {SlurmRunner.__module__}"
+            f" --slurm-runner-spec-path {self.slurm_runner_spec_path}"
+            f" --run-spec-path {run_spec_path}"
+        )
+        # TODO: Make default Slurm arguments configurable.
+        slurm_args: Dict[str, Union[str, int]] = {
+            "account": "nlp",
+            "cpus_per_task": 4,
+            "mem": "32G",
+            "gres": "gpu:0",
+            "open_mode": "append",
+            "partition": "john",
+            "time": "14-0",
+            "mail_type": "END",
+            "job_name": run_name,
+            "output": log_path,
+            "chdir": os.getcwd(),
+        }
+        # TODO: Move resource requirements into RunSpec.
+        if run_spec.name.startswith("msmarco:"):
+            slurm_args["mem"] = "64G"
+        if "device=cuda" in run_spec.name:
+            slurm_args["gres"] = "gpu:1"
+            slurm_args["partition"] = "jag-hi"
```

Comment:
why use `hi`?

Reply:
`jag-important` doesn't seem to exist any more... `jag` is preemptible.